bazel-mode.el

```diff
@@ -89,7 +101,7 @@
   (setq-local comment-start-skip "#+")
   (setq-local comment-end "")
   (setq-local comment-use-syntax t)
-  (setq-local font-lock-defaults '(nil))
+  (setq-local font-lock-defaults bazel-mode--font-lock-defaults)
```

Comment:
Nit: I think it might be a bit more readable to set `font-lock-defaults` to `(list bazel-mode--font-lock-keywords)`, rename the constant appropriately, and remove one level of parentheses from its value.

Reply:
Done.